galleries/examples/scales/symlog_demo.py

```diff
@@ -34,12 +39,85 @@
 plt.show()
 
 # %%
-# It should be noted that the coordinate transform used by ``symlog``
-# has a discontinuous gradient at the transition between its linear
-# and logarithmic regions. The ``asinh`` axis scale is an alternative
-# technique that may avoid visual artifacts caused by these discontinuities.
+# Linear threshold
+# ----------------
+# Since each decade on a logarithmic scale covers the same amount of visual space
+# and there are infinitely many decades between a given number and zero, the symlog
+# scale must deviate from logarithmic mapping in a small range (-x0, x0), so that
+# that range is mapped to a finite visual space.
+#
+# The symlog scale achieves this by defining a parameter *linthresh* and switching
+# to a linear mapping in the region *(-linthresh, linthresh)*.
```

Comment:
x0 isn't defined before this, so perhaps just go right to the *linthresh* name that we use internally instead of adding the next sentence which says basically the same thing.
```suggestion
# scale must deviate from logarithmic mapping in a small range *(-linthresh, linthresh)*, so that
# the range is mapped to a finite visual space.
```